### ISSUE TITLE
Create user programatically with the Management API

### ISSUE THREAD

**Issue by rhino5oh** [auth0/auth0.js#1307]
### Checklist

- [X] I have looked into the [Readme](https://github.com/auth0/auth0.js#readme) and [Examples](https://github.com/auth0/auth0.js/blob/master/EXAMPLES.md), and have not found a suitable solution or answer.
- [X] I have looked into the [API documentation](https://auth0.github.io/auth0.js/index.html) and have not found a suitable solution or answer.
- [X] I have searched the [issues](https://github.com/auth0/auth0.js/issues) and have not found a suitable solution or answer.
- [X] I have searched the [Auth0 Community](https://community.auth0.com) forums and have not found a suitable solution or answer.
- [X] I agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Describe the problem you'd like to have solved

It doesn't look like there is a way to create a user from this SDK.  I'm looking here: https://auth0.github.io/auth0.js/Management.html and see nothing related to creating a new user.

I believe that adding a `createUser` function to this SDK (similar to the `patchUserMetadata` function) would be a great addition.

### Describe the ideal solution

A createUser function for auth0.js

### Alternatives and current workarounds

Manually POST'ing to the management API. Seems strange to hop between manual HTTP POST'ing (for things like creating a user) and using built-in SDK functions (for things like patchUserMetadata)

**Comment by frederikprijck** [comment]
Thanks for reaching out. You should not call those management api endpoints from a public application, so it isn't an endpoint we should add to auth0js.

Instead, you should have your own api that uses our server side SDKs (e.g. node-auth0, auth0.net,... ).

**Comment by rhino5oh** [comment]
Thanks for the info. What you said makes sense the more I am reading. I was definitely caught up a bit on why auth0.js has _some_ management API controls built in, but not all of them.

I have an angular app, which uses NodeJS on the backend. I was certainly getting a bit confused with:

- Management API HTTP endpoints
- auth0-js npm package
- node-aut0 npm package
- new auth0.WebAuth vs checkSession()
- scopes, clients, secrets, tokens, conenctions, ids, etc etc.

Lots and lots to learn here that's for sure! I gave node-auth0 a try on my backend and I was able to create a user. Now to figure out the email provider to send an invite link.
